src/streetscapes/sources/image/mapillary.py

```diff
@@ -77,15 +160,22 @@ def create_session(self) -> requests.Session:
             A `requests` session.
         """
 
-        session = requests.Session()
+        session = Session()
         session.headers.update({"Authorization": f"OAuth {self.token}"})
+        retries = Retry(
+            total=3,
+            backoff_factor=0.1,
+            status_forcelist=[500, 502, 503, 504],
+        )
+        session.mount('https://', HTTPAdapter(max_retries=retries))
         return session
 
-    def fetch_image_ids(
+    def fetch_image_ids_bbox(
         self,
         bbox: list[float],
+        tile_size: float = 0.01,
         fields: list[str] | None = None,
-        limit: int = 500,
+        limit: int = 1000,
```

Comment:
Make additional argument to give specific filename prefix so we can later on distinguish between results from various bbox requests.